src/components/Calculator.js

```diff
@@ -1,105 +1,204 @@
             x
           </button>
         </div>
         <div className="four">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             4
           </button>
         </div>
         <div className="five">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             5
           </button>
         </div>
         <div className="six">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             6
           </button>
         </div>
         <div className="less">
-          <button type="button" className="orange-btn">
+          <button
+            type="button"
+            className="orange-btn"
+            onClick={this.handleButton}
+          >
             -
           </button>
         </div>
         <div className="one">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             1
           </button>
         </div>
         <div className="two">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             2
           </button>
         </div>
         <div className="three">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             3
           </button>
         </div>
         <div className="plus">
-          <button type="button" className="orange-btn">
+          <button
+            type="button"
+            className="orange-btn"
+            onClick={this.handleButton}
+          >
             +
           </button>
         </div>
         <div className="zero">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             0
           </button>
         </div>
         <div className="dot">
-          <button type="button" className="grey-btn">
+          <button
+            type="button"
+            className="grey-btn"
+            onClick={this.handleButton}
+          >
             .
           </button>
         </div>
         <div className="equal">
-          <button type="button" className="orange-btn">
+          <button
+            type="button"
+            className="orange-btn"
+            onClick={this.handleButton}
```

Comment:
- [ ] When I divide any number by zero instead of giving me an error app stops working.

Reply:
Hi, Arslan. Now I'm displaying the error when it happens.